autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -9,14 +9,63 @@
 import scipy.sparse
 
 import sklearn.utils
-from sklearn import preprocessing
+
 from sklearn.base import BaseEstimator
 from sklearn.compose import ColumnTransformer
 from sklearn.exceptions import NotFittedError
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import StandardScaler, OrdinalEncoder
 
 from autoPyTorch.data.base_feature_validator import BaseFeatureValidator, SUPPORTED_FEAT_TYPES
 
 
+def _create_column_transformer(preprocessors: typing.Dict, numerical_columns, categorical_columns):
+    numerical_pipeline = 'drop'
+    categorical_pipeline = 'drop'
+    if len(numerical_columns) > 0:
+        numerical_pipeline = make_pipeline(*preprocessors['numerical'])
+    if len(categorical_columns) > 0:
+        categorical_pipeline = make_pipeline(*preprocessors['categorical'])
+
+    return ColumnTransformer([
+        ('categorical_pipeline', categorical_pipeline, categorical_columns),
+        ('numerical_pipeline', numerical_pipeline, numerical_columns)],
+        remainder='passthrough'
+    )
+
+
+def get_tabular_preprocessors():
```

Comment:
typing
Here as well, I need to know how this function helps.

Reply:
As mentioned, it would be good to include the documentation here too.